apps/typegpu-docs/src/components/ControlPanel.tsx

```diff
@@ -101,17 +106,19 @@ function SelectRow({
         options={options}
         onChange={(newValue) => {
           setValue(newValue);
-          onChange(newValue);
+          runWithCatch(() => onChange(newValue));
         }}
       />
     </>
   );
 }
 
 function ButtonRow({ label, onClick }: { label: string; onClick: () => void }) {
+  const runWithCatch = useSetAtom(runWithCatchAtom);
+
   return (
     <div className="grid h-10 col-span-2">
-      <Button onClick={onClick}>{label}</Button>
+      <Button onClick={() => runWithCatch(() => onClick())}>{label}</Button>
```

Comment:
In this case, since `onClick` takes no arguments, we can pass it directly.
```suggestion
      <Button onClick={() => runWithCatch(onClick)}>{label}</Button>
```